entries.go

```diff
@@ -218,3 +218,13 @@ func GetEntry(bodyByteGetterFunc BodyByteGetter, articleID int) (Item, error) {
 	err = json.Unmarshal(body, &item)
 	return item, err
 }
+
+// ExportEntry queries the API for a specific article content according to the API request /entries/ID
+func ExportEntry(bodyByteGetterFunc BodyByteGetter, articleID int, format string) ([]byte, error) {
+	entryURL := Config.WallabagURL + "/api/entries/" + strconv.Itoa(articleID) + "/export." + format
```

Comment:
I am unsure wether we should check the `format` string for a valid value. I mean, that it is a string, which the API can handle (as of now xml|json|txt|csv|pdf|epub|mobi according to https://app.wallabag.it/api/doc#get--api-entries-{entry}-export.{_format}). Or shall we let the wallabag server handle invalid format strings?

Reply:
I don't think we should. In both case if the `format` is invalid an error will be returned. If we hardcode the formats on the client and a future wallabag version implement new formats the client would require an update, we should avoid that.